### DISCUSSION TITLE
Question: Can I upload any of the art and music to Wikimedia Commons?

### DISCUSSION THREAD

**Discussion by PrincessPandaSource** [FunkinCrew/Funkin#1492] [Q&A] [answered]
Seeing this game is open-sourced, with the source under the Apache License, and the game's README file saying "do whatever you want with the assets", this all makes the game a total treasure trove for Wikimedia Commons to have, or does it? Seeing the album containing the music tracks being sold for money and saying "All Rights Reserved" makes me take precautions.

**Comment by TentaRJ** [comment]
## The creator was referencing that since the game is open source, modifying the game is fully allowed.

### I am just offering my input, but just understand that I am **not** the creator and you **should not** take everything I say seriously.

With that out of the way...

The license does allow for the distribution of all assets (pictures, code, .json charts). I don't know about the songs however since those songs are owned by Kawai Sprite.

### Allowed:
```
 Commercial use
 Modification
 Distribution
 Patent use
 Private use
```
### Not Allowed:
```
 Trademark use
 Liability
 Warranty
```

### I don't know about posting the stuff to Wikimedia Commons however. @ninjamuffin99 is able to answer those questions. I would say hold off on posting to Wikimedia Commons until they respond back if they do.

**Comment by PrincessPandaSource** [comment]
Wikimedia Commons allow any supported media files in which their licenses allow commercial use and creation of derivative works. Images of trademarked material are allowed if they are under such a free license. However, I do agree it's best to receive an official answer from the creator before uploading the art or music to Wikimedia Commons.

**Comment by ninjamuffin99** [comment]
Stuff with the ASSETS we a little loose with. I didn't personally make the art, so I don't exactly want to speak on it RIGHT NOW.

It's somethin I'll have to clear up, but for now I'll say the current license we have on the Github is for the CODE and all that FOR NOW. Music and art and all that we need to work out like an ACTUAL license or whatever and figure out what we want to do with it. It was moreso for maybe people who wanted to make fan media and all that. Fangames, remixes of music, youtube thumbnails, etc. etc. etc. like some gentleman's agreement type deal. Having a broad LICENSE is somethin I don't want to be TOO willy nilly with, make sure it all fits with what we're fine and comfortable with!

**Comment by PrincessPandaSource** [comment]
So, should I hold off uploading the assets and request deletion of [the image of the logo](https://commons.wikimedia.org/wiki/File:Friday_Night_Funkin%27.png) uploaded on there already?

**Comment by ninjamuffin99** [comment, marked as answer]
Assets can be uploaded to Wikipedia! Just I'm not sure if theyre gonna to be under the wikimedia license. Like idk what wikipedia requires and all that. BUT WIKIPEDIA IS ALLOWED TO USE ANYTHING FROM FUNKIN

**Comment by TentaRJ** [comment]
> Assets can be uploaded to Wikipedia! Just I'm not sure if theyre gonna to be under the wikimedia license. Like idk what wikipedia requires and all that. BUT WIKIPEDIA IS ALLOWED TO USE ANYTHING FROM FUNKIN

Oh I see. I interpreted the other thing wrong. My apologies.

**Comment by PrincessPandaSource** [comment]
Like under fair use? I'm just wondering if they can be uploaded to Wikimedia Commons, which is supposed to be a repository for _free media anyone can use_, not just Wikipedia? If you're confused, read [this article](https://commons.wikimedia.org/wiki/Commons:Licensing). (To avoid stepping into mines, I have to be really sure before I proceed to upload some of the assets or excerpts.)
